# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/media/webrtc/jsapi/PeerConnectionImpl.h
# Commit: 88a9fe04aaa5
# Full Hash: 88a9fe04aaa582676a84734d67c9069b68a7f3e1
# Author: Andreas Pehrson <apehrson@mozilla.com>
# Date: 2021-11-01 18:17:33
# Regressor Bug: 1654112
# File Overlap Count: 5
# Description:
#   Bug 1654112 - Update Audio- and VideoConduit stats API and integration. r=ng,bwc
#   
#   For AudioConduit this patch:
#   - Uses correct threads per upstream, pending changes due to the threading model
#     update that's coming
# ==============================================================================

diff -r 3eae9f9279f6 -r 88a9fe04aaa5 dom/media/webrtc/jsapi/PeerConnectionImpl.h
--- a/dom/media/webrtc/jsapi/PeerConnectionImpl.h	Fri Jan 29 15:38:54 2021 +0100
+++ b/dom/media/webrtc/jsapi/PeerConnectionImpl.h	Mon Jan 18 15:28:29 2021 +0100
@@ -448,9 +448,7 @@
   PeerConnectionImpl(const PeerConnectionImpl& rhs);
   PeerConnectionImpl& operator=(PeerConnectionImpl);
 
-  RefPtr<dom::RTCStatsPromise> GetReceiverStats(
-      const RefPtr<MediaPipelineReceive>& aPipeline);
-  RefPtr<dom::RTCStatsPromise> GetSenderStats(
+  nsTArray<RefPtr<dom::RTCStatsPromise>> GetSenderStats(
       const RefPtr<MediaPipelineTransmit>& aPipeline);
   RefPtr<dom::RTCStatsPromise> GetDataChannelStats(
       const RefPtr<DataChannelConnection>& aDataChannelConnection,